.pre-commit-config.yaml

```diff
@@ -1,10 +1,36 @@
 repos:
+
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.6.0
+  hooks:
+    # Checks python syntax
+    - id: check-ast
+    - id: check-case-conflict
+    - id: check-merge-conflict
+    - id: end-of-file-fixer
+    - id: trailing-whitespace
+
 - repo: https://github.com/astral-sh/ruff-pre-commit
-  # Ruff version.
-  rev: v0.4.10
+  # Matches Ruff version in requirements.
+  rev: v0.5.0
   hooks:
-    # Run the linter.
     - id: ruff
+      name: lint with ruff
       args: [ --fix ]
-    # Run the formatter.
     - id: ruff-format
```

Comment:
I think this is too drastic when formatting isn't applied to the code base first. I just tried to commit speech.py but it reformats the whole file.

Reply:
I'll be performing a full `ruff format` in another commit soon